dockers/docker-platform-monitor/docker_init.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Generate supervisord config file and the start.sh scripts
+mkdir -p /etc/supervisor/conf.d/
+
+if [ -e /usr/share/sonic/platform/pmon_daemon_control.json ]; then
+sonic-cfggen -j /usr/share/sonic/platform/pmon_daemon_control.json -t /usr/share/sonic/templates/docker-pmon.supervisord.conf.j2 > /etc/supervisor/conf.d/supervisord.conf
```

Comment:
Is there a reason why you chose not to indent the lines inside this if/else statement?

Reply:
no reason... fixed.